themes/default/content/docs/intro/concepts/resources/_index.md

```diff
@@ -28,13 +28,13 @@ menu:
         "#version": "/docs/intro/concepts/resources/options/version",
         "#components": "/docs/intro/concepts/resources/components",
         "#authoring-a-new-component-resource": "/docs/intro/concepts/resources/components/#authoring-a-new-component-resource",
-        "#creating-a-child-resource": "/docs/intro/concepts/resources/components/#creating-a-child-resource",
+        "#creating-child-resources": "/docs/intro/concepts/resources/components/#creating-child-resources",
```

Comment:
The [previous version of this page](https://raw.githubusercontent.com/pulumi/pulumi-hugo/12c547508213699a93cde3aae460f610e21d6a24/themes/default/content/docs/intro/concepts/resources.md) used the heading `Creating Child Resources`, so the resulting anchor tag would've been `creating-child-resources`.